pandora-client-web/src/components/chatroomAdmin/chatroomAdmin.scss

```
@@ -101,6 +128,7 @@
 		}
 
 		.dropdown-content {
+			@include flex(column);
 			display: none;
 			position: absolute;
```

This element can easily overflow the screen for long lists. The suggested solution isn't perfect, but solves most of the overflowing:
```suggestion
			position: absolute;
			max-height: 50vh;
			overflow-y: auto;
```